### ISSUE TITLE
[UI] Dictionary - display very long name

### ISSUE THREAD

**Issue by ghoshted** [ada-discovery/ada-issues#200]
![image](https://user-images.githubusercontent.com/509036/141821866-0ca810fb-ae13-449f-abb0-20a18f73c1d9.png)

to 

![image](https://user-images.githubusercontent.com/509036/141825916-3c74c8f3-fe58-498b-b00a-8216fc5b8d6e.png)

I injected ..
```
td {
  max-width: 300px;
  overflow: hidden;
  display: table-cell;
  overflow-wrap: anywhere;
}
```

**Comment by sherzinger** [comment]
Thank you for provided the report and resolution in the same issue! :-)

**Comment by LBolzani** [comment]
@soumyabrataghosh the update is available at https://10.240.16.149/

**Comment by ghoshted** [comment]
![image](https://user-images.githubusercontent.com/509036/155130898-9d32c5da-6039-4589-af9e-162302ef9ace.png)
Perfect!
